text/utilities/0000-health-checker-buildpack.md

```diff
@@ -36,66 +36,62 @@ When conditions are met, the buildpack will do the following:
 
 * Contributes a health checker to a layer marked `launch` with command on `$PATH`
 * Contributes a process type called `health-check` which will execute the installed health checker.
+  * If any known environment variables for [supported health checkers](#default-health-checker) are set at build time, they will be set as process specific environment variables on the `health-check` process type at the `.default` level.
 
 The Health Checker buildpack will optionally be added to the language family buildpacks that wish to include support for it. The primary targets will be the Java and Rust buildpacks. The buildpack should be added reasonably close to the end of an order group for a given language family.
 
-Other buildpacks in the order group may influence the configuration of the the health checker by modifying the `BP_HEALTH_CHECKER_*` environment variables. It is recommended that buildpacks use the `.default` setting when doing this so that any user specified options will have the highest precedence.
+Other buildpacks in the order group may influence the configuration of the the health checker by modifying the `BP_HEALTH_CHECKER_DEPENDENCY` or health checker specific environment variables. It is recommended that buildpacks use the `.default` setting when doing this so that any user specified options will have the highest precedence.
```

Comment:
fwiw at build time (so for `BP_HEALTH_CHECK_DEPENDENCY`) user provided env vars always win. But I still like the strategy of recommending `.default` aesthetically

At runtime `.default` vs `.override` will matter.